arduino-iot-cloud.js

```diff
@@ -82,9 +82,26 @@ module.exports = function (RED) {
               this.thing = config.thing;
               this.propertyId = config.property;
               this.propertyName = config.name;
+
+              //console.log("Node Constructor: ", config.name);
+              try {
```

Comment:
As you already noted this block fails if there isn't a connection at startup. Probably another option is to add a select input in the node configuration phase and let the user select a device_id. In that case, the device id would be stored in an internal variable and not fetched at startup. The cons is that in case the device is detached the node must be reconfigured.